### DISCUSSION TITLE
ZX Spectrum appears in Home Computers section on a fresh install

### DISCUSSION THREAD

**Discussion by zugurudumba** [Rparadise-Team/Koriki#6] [Q&A]
Does anyone know why ZX Spectrum appears alongside the Home Computers section on a fresh install, even if there are no ZX Spectrum roms on the SD card?

I had the same bug on fresh installs of Adam image on RG350M.

**Comment by eduardofilo** [comment]
It's not a bug, it's a feature :-)
SimpleMenu by default, shows the first system defined in each group when all the systems of that group have empty directories. That way it have the opportunity to display the warning that there are no ROMs compatible with the extensions defined in those systems.

**Reply by eduardofilo** [reply]
Install some ROMs in whatever system of Home Computers group, and you will get rid of the ZX Spectrum system (unless you install ROMs from that system).

**Reply by zugurudumba** [reply]
Noted, I'll try this. Thanks!
